packages/daimo-api/src/api/getLinkStatus.ts

```diff
@@ -87,6 +88,26 @@ export async function getLinkStatus(
       return ret;
     }
 
+    case "notev2": {
+      const sender = await nameReg.getEAccountFromStr(link.sender);
+      if (sender == null) {
+        throw new Error(`Note sender not found: ${link.sender}`);
+      }
+      const ret = noteIndexer.getNoteStatusBySeq(sender.addr, link.seq);
+      if (ret == null) {
+        const pending: DaimoNoteStatus = {
+          status: "pending",
+          ephemeralOwner: undefined,
+          link,
+          sender,
+          seq: link.seq,
+          dollars: link.dollars,
+        };
+        return pending;
+      }
+      return ret;
```

Comment:
add a check here that logs an error if `ret.seq !== link.seq`

Reply:
deleted seq